### PR TITLE
Adding tags associated with a task

### PR DESCRIPTION
**Pull request by dpick**
This adds the tags field on tasks to the pull.

### CODE REVIEW

**Comment by cmerrick** [comment]
Hi @dpick, thanks for your contribution!

In order for us to evaluate and accept your PR, we ask that you [sign a contribution license agreement](https://stitch-cla-enforcer.herokuapp.com/). It's all electronic and will take just minutes.

**Comment by cmerrick** [comment]
You did it @dpick!

Thank you for signing the Singer Contribution License Agreement.

**Comment by DenGrishin** [comment]
Hey guys, 
Could you please review these changes. 
We found it very helpful.
Thank you.

**Comment by MattBurlew** [comment]
This would be super helpful for us as well.

**Comment by ghost** [comment]
This is very helpful, please, review it.

File: tap_asana/schemas/tasks.json [hunk 1]
```diff
@@ -195,6 +195,11 @@
         "string"
       ]
     },
+    "tags": {
+      "type": [
+        "null",
+        "array"
+      ],
```
Comment (jacobrobertbaca) [inline tap_asana/schemas/tasks.json:202]: It's unclear what the schema is for the entries in the array?  Take a look at #7 to see what the addition of an array field should look like -- you need to specify the schema of the items in the array.

Also, this is missing a closing curly brace for the `tags` field

**Review by jacobrobertbaca — CHANGES_REQUESTED** [review]

**Comment by nick-mccoy** [comment]
We incorporated this change into #10.  Closing this

**Pull request closed without merging**